index.html

```diff
@@ -253,6 +253,100 @@ <h1>Alert</h1>
       </div>
       <!-- alert end -->
       <!-- button start -->
+    <div class="container">
+      <h1>Solid Buttons</h1>
+      <p>Here we created Buttons.</p>
+      <p>
+        The button component has six variants alert-primary,alert-secondary, alert-success, alert-danger, alert-warning and alert-info.
+      </p>
+      <div class="inline">
+        <div class="btntype" >
+          <button class="btn btn-primary">Primary</button>
+         <button class="btn btn-secondary">Secondary</button>
+         <button class="btn btn-success">Success</button>
+         <button class="btn btn-danger">Danger</button>
+         <button class="btn btn-warning">warning</button>
+         <button class="btn btn-info"info> info</button>
+
+
+        </div>
+
+      </div>
+
+    </div>
+    <div class="code-container">
+      <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=panda-syntax&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=12px&lh=146%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-primary%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520Primary%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-secondary%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520Secondary%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-success%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520Success%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-danger%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520Danger%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-warning%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520warning%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-info%2522info%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520info%253C%252Fbutton%253E"
+ 
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+    </div>
+    <div class="container">
+      <h1>Outline Buttons and Link Buttons</h1>
+      <p>Here we created Outline and link Buttons.</p>
+      <p>
+        The button component has three variants primary,secondary,  danger, .
+      </p>
+      <div class="inline">
+        <div class="btntype" >
+          <button class="btn btn-outline-primary">Primary</button>
+        <button class="btn btn-outline-secondary">Secondary</button>
+    
+        <button class="btn btn-outline-danger">Danger</button>
+        <a target="_blank" class="fcc-btn" href="#">neoGCamp</a>
+        <a class="btn btn-primary" href="#">neoGCamp</a>
+  
+
+
+        </div>
+
+      </div>
+
```

Comment:
remove all unnecessary spaces in this file.

Reply:
+1

Reply:
updated anshul thank you